zeppelin-server/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java

```diff
@@ -483,4 +483,149 @@ public void testEditOnDoubleClick() throws Exception {
       handleException("Exception in ParagraphActionsIT while testEditOnDoubleClick ", e);
     }
   }
+
+  @Test
+  public void testSingleDynamicFormTextInput() throws Exception {
+    if (!endToEndTestEnabled()) {
+      return;
+    }
+    try {
+      createNewNote();
+
+      setTextOfParagraph(1, "%spark println(\"Hello \"+z.input(\"name\", \"world\")) ");
+
+      runParagraph(1);
+      waitForParagraph(1, "FINISHED");
+      collector.checkThat("Output text is equal to value specified initially",
+              driver.findElement(By.xpath(getParagraphXPath(1) + "//div[contains(@class, 'text plainTextContent')]")).getText(),
+              CoreMatchers.equalTo("Hello world"));
+
+      driver.findElement(By.xpath(getParagraphXPath(1) + "//input")).clear();
+      driver.findElement(By.xpath(getParagraphXPath(1) + "//input")).sendKeys("Zeppelin");
+
+      collector.checkThat("After new data in text input form, output should not change",
```

Comment:
Thanks @AhyoungRyu I went ahead and checked the other test case messages and fixed an extra white space I found.